app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -277,32 +253,32 @@ def process_hardware(template_object, params, metadata, domain)
   def default_value(params, name)
     value = nil
     params.each do |param|
-      if param.name == name
-        value = param.value
+      if param[:name] == name
+        value = param[:value]
```

Comment:
maybe this method can be something like:

name_param = params.detect { |param| param[:name] == name }
named_param[:value]

although to tell the truth I do not understand exactly what this is supposed to do, since it seems like it might be overwriting value several times.

Reply:
will check whether you solution would work